src/main/java/mocacong/server/controller/MemberController.java

```diff
@@ -19,11 +17,21 @@
 public class MemberController {
 
     private final MemberService memberService;
+    private final JwtTokenProvider jwtTokenProvider;
 
     @Operation(summary = "회원가입")
     @PostMapping
     public ResponseEntity<MemberSignUpResponse> signUp(@RequestBody MemberSignUpRequest request) {
         MemberSignUpResponse response = memberService.signUp(request);
         return ResponseEntity.ok(response);
     }
+
+    @Operation(summary = "회원탈퇴")
+    @DeleteMapping
+    public ResponseEntity<String> delete(@RequestHeader("Authorization") String authorizationHeader) {
```

Comment:
`@SecurityRequirement(name = "JWT")` 어노테이션이 있어야 swagger에서 인증 없이 해당 api를 쏠 때 에러를 띄워줍니다.

```suggestion
    @Operation(summary = "회원탈퇴")
    @SecurityRequirement(name = "JWT")
    @DeleteMapping
    public ResponseEntity<String> delete(@RequestHeader("Authorization") String authorizationHeader) {
```

Reply:
알려주셔서 감사합니다!